drivers/sensors/cirque_pinnacle.c

```diff
@@ -200,39 +198,86 @@ void cirque_pinnacle_init(void) {
 #endif
 
     touchpad_init = true;
+
     // Host clears SW_CC flag
     cirque_pinnacle_clear_flags();
 
-    // Host configures bits of registers 0x03 and 0x05
-    RAP_Write(SYSCONFIG_1, SYSCONFIG_1_VALUE);
-    RAP_Write(FEEDCONFIG_2, FEEDCONFIG_2_VALUE);
-
-    // Host enables preferred output mode (absolute)
-    RAP_Write(FEEDCONFIG_1, FEEDCONFIG_1_VALUE);
+    // SysConfig1 (Low Power Mode)
+    // Bit 0: Reset, 1=Reset
+    // Bit 1: Shutdown, 1=Shutdown, 0=Active
+    // Bit 2: Sleep Enable, 1=low power mode, 0=normal mode
+    // send a RESET command now, in case QMK had a soft-reset without a power cycle
+    RAP_Write(SYSCONFIG_1, 0x01);
+    wait_ms(20); // Pinnacle needs 10-15ms to boot, so wait long enough before configuring
```

Comment:
20ms doesn't work for me, 21 and up does. Up this to 25~30 to be safe?
```suggestion
    wait_ms(30); // Pinnacle needs 10-15ms to boot, so wait long enough before configuring
```